onboard/catkin_ws/src/data_pub/README.md

```diff
@@ -65,3 +63,9 @@ The `dvl_raw` script publishes the raw DVL data using the `dvl_raw` message from
 The `dvl_to_odom` script converts the raw dvl data to an `odometry` message for use in other scripts. It publishes to topic `sensors/dvl/odom`.
 
 You can launch both scripts using the `pub_dvl.launch` file.
+
+## Pressure Sensor Documentation
+
+The Blue Robotics pressure sensor sends raw serial data to the main computer. This data is then published to the `/sensors/depth` topic. Note that this is a separate package and will need to be run in addition to this `offboard_comms` package. The data is of type `nav_msgs/Odometry.msg`. The data is published by the `data_pub` node.
```

Comment:
Change the wording of this line since it is part of the `data_pub` README, not `offboard_comms`.